tgui/packages/tgui/interfaces/AntagInfoBlob.tsx

```diff
@@ -156,37 +155,39 @@ const Minions = (props) => {
 
 const Structures = (props) => {
   return (
-    <Collapsible title="Structures">
+    <Collapsible title="Сооружения">
       <Box>
-        Normal Blobs will expand your reach and can be upgraded into special
-        blobs that perform certain functions. Bear in mind that expanding into
-        space has an 80% chance of failing!
+        Обычные клетки блоба расширят ваши границы и могут быть улучшены до
+        специальных клеток блоба, которые выполняют определенные функции. Имейте
+        в виду, что расширение в космос с вероятностью 80% не удастся!
       </Box>
       <br />
-      <Box>You can upgrade normal blobs into the following types of blob:</Box>
+      <Box>Вы можете превратить обычные клетки блоба в следующие типы:</Box>
       <Divider />
       <LabeledList>
-        <LabeledList.Item label="Strong Blobs">
-          Strong blobs are expensive but take more damage. In additon, they are
-          fireproof and can block air, use these to protect yourself from
-          station fires.
+        <LabeledList.Item label="Сильные клетки блоба">
+          Сильные клетки блоба стоят дороже, но наносят больше урона. Кроме
+          того, они огнеупорные и могут блокировать воздух, используйте их,
+          чтобы защититься от пожаров на станции.
         </LabeledList.Item>
-        <LabeledList.Item label="Reflective Blobs">
-          Upgrading strong blobs creates reflective blobs, capable of reflecting
-          most projectiles at the cost of the strong blob&apos;s extra health.
+        <LabeledList.Item label="Отражающие клетки блоба">
+          Улучшение сильных клеток блоба создает отражающие клетки блоба,
+          способные отражать большинство снарядов за счет дополнительного
+          здоровья сильной клетки блоба.
```

Comment:
Лишние строки не надо добавлять
```suggestion
        <LabeledList.Item label="Отражающий блоб">
          Улучшение сильного блоба создает блоба, отражающего
          большинство снарядов, ценой увеличенного здоровья.
```